rllib/algorithms/algorithm.py

```diff
@@ -151,8 +150,8 @@ class Algorithm(Trainable):
     Algorithms contain a WorkerSet under `self.workers`. A WorkerSet is
     normally composed of a single local worker
     (self.workers.local_worker()), used to compute and apply learning updates,
-    and optionally one or more remote workers (self.workers.remote_workers()),
-    used to generate environment samples in parallel.
+    and optionally one or more remote workers  used to generate environment
```

Comment:
typo: two spaces

Reply:
Can we describe here how these remote workers can now be accessed? Via their ID? Or not at all?

Reply:
done, makes sense.